include/glow/Base/Tensor.h

```diff
@@ -551,6 +551,22 @@ class Tensor final {
   /// Dump a textual representation of the shape of this Tensor to std::string.
   std::string getShapeToString() const;
 
+  /// Dump the data content of the tensor to a raw-binary file.
+  void dumpToRawBinaryFile(const char *filename);
```

Comment:
StringRef instead of const char *?

Reply:
Where do you invoke these methods? I see only examples in tests.

Reply:
The tensor loading methods (binary/text) are used in ModelProfiler.cpp for feeding the model input placeholders with data during the profiling precedure. The methods were designed to provide an easy method for the user to feed with data potentially generated in other places (a Python script, a C code):
- the binary loading method expects a blob which has the exact tensor size in bytes
- the text loading method expects a single array of comma separated values
Having these methods to have a more specific format (Glow`ish) by also including meta-information (like tensor type/shape) does not sound easy for user to generate externally.
I did not need the dump methods but I wrote them to match the loading methods.

Reply:
Replaced const char * with llvm::StringRef.